### PR TITLE
puppeteer tests refactor / multiglobe fix

### PR DESCRIPTION
**Pull request by peppsac**
This PR started a simple bugfix for the `multiglobe.html` example (see 2nd commit).

But when I tried to add a test to make sure the bug won't come back I faced an issue in the puppeteer tests...

This PR refactors the `test-examples` code (1st commit) to ease test writing and limit the boilerplate needed. The 2nd commit fixes and tests the initial issue

### CODE REVIEW

File: test/examples/bootstrap.js [hunk 1]
```diff
@@ -140,6 +149,43 @@ before(async () => {
...
+                        resolve(true);
+                    }
+                }
+                v.mainLoop.addEventListener('command-queue-empty', resolveWhenReady);
+            });
+        }));
+
+        await waitNextRender(page);
+
+        if (process.env.SCREENSHOT_FOLDER) {
```
Comment (tbroyer) [inline test/examples/bootstrap.js]: Refactor into a helper function rather than duplicate between `loadExample` and `waitUntilItownsIsIdle`?

Comment (peppsac) [reply test/examples/bootstrap.js]: Good idea, done.

File: test/examples/bootstrap.js [hunk 2]
```diff
@@ -76,9 +85,22 @@ before(async () => {
...
+        // eslint-disable-next-line no-param-reassign
+        page.loadExampleCalled = true;
+        page.setViewport({ width: 400, height: 300 });
+
+        await page.goto(url);
+        await page.waitFor('#viewerDiv > canvas');
+
+        // install a globally available __getView helper
+        await page.evaluate(() => {
+            window.__getView = function _() {
```
Comment (zarov) [inline test/examples/bootstrap.js:103]: This could be removed later if we have stricter example rules, WDYT ?

**Review by zarov — APPROVED** [review]
LGTM, thanks for synthesizing all these things.

**Comment by zarov** [comment]
Don't forget to squash !

**Pull request merged**